logos/brand-guidelines.md

```diff
@@ -0,0 +1,80 @@
+# Resources and Guidelines for TiDB Community
+
+These resources and guidelines were designed to assist the TiDB Community to effectively and consistently build and protect the community’s brand 
+through visual communication.
+
+# TiDB Logo
+
+## Seal+Signature(formal)
```

Comment:
Leave a whitespace before the parenthesis.

Pls also update the following ones.